doc/user/content/cloud/security.md

```diff
@@ -18,3 +18,74 @@ Materialize Cloud secures your connections by enforcing that clients connect via
 * Single sign-on integration (SSO)
 
 In the future, the enterprise product will also provide secure network ingress and egress. If you have other specific security requirements, please [let us know](../support).
+
+## Static IP addresses
+
+All Materialize [Cloud deployments](../cloud-deployments/) come with a static IP address.
+
+This gives you the ability to connect your Materialize Cloud deployments to [sources](../../sql/create-source) and [sinks](../../sql/create-sink/) secured with a firewall.
+
+Allowing the static IP address enables the connection from your Materialize Cloud deployments to your sources and sinks.
+
+The specific commands to allow the static IP address will vary depending on your operating system and firewall (e.g. `iptables`, `firewall-cmd`, UFW, AWS, Azure, and etc). Please refer to the appropriate documentation for your specific firewall.
+
+### Getting the Static IP Address
+
+Once you’ve created a [deployment](../cloud-deployments/), to get the static IP address associated with your deployment you can follow these steps:
+
+1. Click on a deployment to see the individual deployment view.
+2. On the right-hand side, click on the copy button to copy the IP address.
+
+### Allowing the Static IP Address
+
+The following is a list of commands to allow the static IP address with some of the most common firewalls.
+
+#### IPTABLES
```

Comment:
Could these be cased as whatever the original casing is (_e.g._ iptables), instead of defaulting to upper case?